app/views/main_window.py

```diff
@@ -569,7 +572,7 @@ def clone_essential_paths(
             )
         )
         # Sanitize the input so that it does not produce any KeyError down the road
-        new_instance_name = self.__ask_for_new_instance_name().strip()
```

Comment:
You shouldn't be just removing the .strip(), then you're not sanitizing it

Fix the underlying issue.

Reply:
it is 
def __ask_for_new_instance_name(self) -> str:
        instance_name, ok = show_dialogue_input(
            title="Create new instance",
            text="Input a unique name of new instance that is not already used:",
        )
        if ok:
            return instance_name.strip()
        else:
            return None